docusaurus/docs/React/contexts/channel-state-context.mdx

```diff
@@ -67,12 +67,20 @@ Error object (if any) in loading the `channel`, otherwise null.
 
 ### giphyVersion
 
-The giphy version to use when displaying giphies. 
+The giphy version to use when displaying giphies.
 
-| Type                                                                                                                                                |
-| --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Type        |
+| ----------- | -------------- | -------------------- | -------------------------- | ------------- | ------------------- | -------------------------- |
```

Comment:
This change should be reverted. The auto-formatter does not take into consideration the escape chars in front of the pipe (`\|`) char. The result then does not display the pipe char:

![image](https://user-images.githubusercontent.com/32706194/190636316-fcd65ec8-3091-45d2-81d2-6a3f7fe7f387.png)